app/src/main/res/layout-sw600dp/administrator_controls_general_view.xml

```diff
@@ -14,36 +14,34 @@
     android:layout_height="wrap_content">
 
     <TextView
+      style="@style/Heading5"
       android:id="@+id/general_text_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
```

Comment:
You can remove this too because `android:layout_height="wrap_content"` is already mentioned in `Heading5 -> Common1`

Reply:
Done.